project/AppDependencies.scala

```diff
@@ -6,7 +6,7 @@ object AppDependencies {
   lazy val compile = Seq(
     "uk.gov.hmrc" %% "bootstrap-play-25" % "3.14.0",
     "uk.gov.hmrc" %% "play-json-union-formatter" % "1.4.0",
-    "uk.gov.hmrc" %% "play-reactivemongo" % "6.2.0",
+    "uk.gov.hmrc" %% "simple-reactivemongo" % "7.3.0-play-25",
```

Comment:
is this wanted?

Reply:
Yes for `findAndUpdate`

Reply:
I believe I have used `findAndUpdate` with `play-reactivemongo`

Reply:
It wasn't there in that version we had originally anyway :(

Reply:
https://github.com/hmrc/simple-reactivemongo this seems to be the HMRC version

Reply:
You can also use `findAndModify` which is basically the same as `findAndUpdate`

Reply:
`play-reactivemongo` doesnt seem to be being supported any longer (no releases since May) whilst `simple-reactivemongo` is actively worked on within HMRC